spec/query_spec.cr

```diff
@@ -189,3 +217,22 @@ private def insert_a_user(name = "Paul", age = 34)
       Time.now
   end
 end
+
+private def insert_a_post(title = "Fast as C")
+  LuckyRecord::Repo.run do |db|
+    db.exec "INSERT INTO posts(title, created_at, updated_at) VALUES ($1, $2, $3)",
+      title,
+      Time.now,
+      Time.now
+  end
+end
+
+private def insert_a_comment(post_id, body = "Slick as Ruby")
+  LuckyRecord::Repo.run do |db|
+    db.exec "INSERT INTO comments(post_id, body, created_at, updated_at) VALUES ($1, $2, $3, $4)",
+      post_id,
+      body,
+      Time.now,
+      Time.now
+  end
+end
```

Comment:
You can now use "boxes" which will handle most of this for you: https://github.com/luckyframework/lucky_record/tree/master/spec/support/boxes

See it in action here: https://github.com/luckyframework/lucky_record/pull/135/files#diff-5085ddc121b47697c6b81979a09f077fR19

Reply:
I forgot about boxes, they're very convenient.